src/entry-client.js

```diff
@@ -139,22 +138,9 @@ export default function initializeClient(createApp, clientOpts) {
     enablePerfMarks = opts.enablePerfMarks;
 
     const { shouldProcessRouteUpdateDefaults: spruDefaults } = opts;
-    let { initialState } = opts;
-
-    if (isString(opts.initialStateMetaTag)) {
-        if (initialState) {
-            opts.logger.error('initialState and initialStateMetaTag should not be used together');
-        }
-        try {
-            const meta = document.querySelector(`meta[name="${opts.initialStateMetaTag}"]`);
-            initialState = JSON.parse(meta.getAttribute('content'));
-        } catch (e) {
-            opts.logger.error(`Error parsing meta tag content: ${opts.initialStateMetaTag}`);
-        }
-    }
```

Comment:
Breaking change - no longer using this and instead relying on the application to use the `opts.initialState` approach